cedar-policy-core/src/est/expr.rs

```diff
@@ -655,6 +655,7 @@ impl Expr {
                 in_expr,
             }) => ast::Name::from_normalized_str(entity_type.as_str())
                 .map_err(FromJsonError::InvalidEntityType)
+                .map(ast::EntityType::from)
```

Comment:
Should this instead by `ast::EntityType::from_normalized_str()` and then avoid the `.map()`?